app/assets/javascripts/stimulus/loaders/autoloader.js

```diff
@@ -13,13 +37,16 @@ function queryControllerNamesWithin(element) {
 }
 
 function extractControllerNamesFrom(element) {
-  return element.getAttribute(controllerAttribute).split(/\s+/).filter(content => content.length)
+  const tokenList = createTokenList(element, controllerAttribute)
+  return Array.from(tokenList).map(name => ({ element, name }))
```

Comment:
Maybe storing the currently autoloader identifiers on the `html`, `body`, or a `<meta>` element would suffice, instead of on each individual element.

Reply:
What are your thoughts around placing it on `html`, `body` or `meta`?  I was thinking that with a `data-stimulus-autoloading` attribute on the element directly you could at least target that with a css class and have a path to display a loading indicator that way (or block interaction).